libvips/iofuncs/util.c

```diff
@@ -1637,7 +1653,13 @@ vips__temp_name(const char *format)
 
 	g_snprintf(file, FILENAME_MAX, "vips-%d-%u", serial, g_random_int());
 	g_snprintf(file2, FILENAME_MAX, format, file);
-	name = g_build_filename(vips__temp_dir(), file2, NULL);
+
+	/* We can't use g_build_filename() since that will use `\` on win, and
+	 * mixed / and \ directory separators seems to confuse g_spawn_*().
+	 */
+	char *tmpd = vips__temp_dir();
+	name = g_strdup_printf("%s/%s", tmpd, file2);
```

Comment:
This might be a bit neater:
```suggestion
	name = g_build_path("/", tmpd, file2, NULL);
```